### DISCUSSION TITLE
test dataset is broken

### DISCUSSION THREAD

**Discussion by Ryray** [Congyuwang/AlphaNetV3#14] [Q&A] [answered]
Cannot unzip test dataset. Shows that the dataset is borken

**Comment by Congyuwang** [comment]
Right. The dataset is broken. Sorry for that.

**Comment by Ryray** [comment]
Could you please upload the test data again? thx!

**Comment by Congyuwang** [comment]
Ok, I'll do it after the holiday.

**Comment by Ryray** [comment]
thx man!

**Comment by Congyuwang** [comment, marked as answer]
I just checked. This zip file is not broken. It is generated by `pandas.to_csv("xxx.zip")`. Don't try to decompress it with decompression software. Do this:

```python
import pandas as pd

pd.read_csv("test_data.zip")
```

**Comment by Ryray** [comment]
That works man! thx!
